addons/hr_presence/security/sms_security.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="ir_rule_sms_template_hr_manager" model="ir.rule">
+        <field name="name">SMS Template: hr manager CRUD on employee templates</field>
+        <field name="model_id" ref="sms.model_sms_template"/>
+        <field name="groups" eval="[(4, ref('hr.group_hr_manager'))]"/>
```

Comment:
You are adding an ir rule for hr manager, but I cannot find any acls for them. Am I missing something ?

Reply:
HR Manager is supposed to have access to edit this sms template, no? Is it what you are asking?

https://github.com/odoo/odoo/blob/4db8e0260c30d616371bb4159e470f43a1d61233/addons/hr_presence/data/sms_data.xml#L4-L10

Reply:
You did not give them any ACLs. They currently have no create/write/unlink right. Read is granted to everyone on all templates so nothing to do for that one. So either you forgot ACLs, either this change is unnecessary.

Reply:
Indeed, something is wrong with ACLs

(I read acls in your first message as acts :laughing: )